src/topology/category/Profinite.lean

```diff
@@ -154,4 +154,24 @@ monadic_creates_limits _
 instance Profinite.has_limits : limits.has_limits Profinite :=
 has_limits_of_has_limits_creates_limits Profinite_to_Top
 
+/-- Any morphism of profinite spaces is a closed map. -/
+lemma Profinite.is_closed_map {X Y : Profinite} (f : X ⟶ Y) : is_closed_map f :=
+show is_closed_map (Profinite.to_CompHaus.map f), from CompHaus.is_closed_map _
+
+/-- Any continuous bijection of profinite spaces is an isomorphism. -/
+lemma Profinite.is_iso_of_bijective {X Y : Profinite} (f : X ⟶ Y)
+  (bij : function.bijective f) : is_iso f :=
+begin
+  haveI := CompHaus.is_iso_of_bijective (Profinite.to_CompHaus.map f) bij,
+  exact is_iso_of_fully_faithful Profinite.to_CompHaus _
+end
+
+/-- Any continuous bijection of profinite spaces is an isomorphism. -/
+noncomputable def Profinite.iso_of_bijective {X Y : Profinite} (f : X ⟶ Y)
+  (bij : function.bijective f) : X ≅ Y :=
+by letI := Profinite.is_iso_of_bijective f bij; exact as_iso f
+
+instance Profinite.forget_reflects_isomorphisms : reflects_isomorphisms (forget Profinite) :=
+⟨by introsI A B f hf; exact Profinite.is_iso_of_bijective _ ((is_iso_iff_bijective ⇑f).mp hf)⟩
+
```

Comment:
This all looks great! Perhaps these could be moved up into the `Profinite` namespace (and `Profinite_to_Top` could go in that namespace too, with the name `Profinite.to_Top`)?

Reply:
done!